File: MOST_MMC_Client/MOST_JPB/app/model/codes/SearchCountryCodeParm.js
Ext.define('MOST.model.codes.SearchCountryCodeParm',{
	extend: 'MOST.model.foundation.parm.BizParm',
	fields:[{
		name:'cntryCd',
		type:'string'
	},{
		name:'cntryNm',
		type:'string'
	},{
		name:'isEuro',
		type: 'string'
	},{
		name:'flagState',
		type:'string'
	},{
		name: 'updatedBy',
		type: 'string'
	},{
		name: 'updatedTime',
		type: 'date',
		convert:function(value){
			var d = null;
			if (value){
				d = new Date();
				d.setTime(value);
			}
			return d;
		}
	}]
});

File: MOST_MMC_BaseBizComponent/src/com/tsb/most/basebiz/parm/codes/SearchCountryCodeParm.java
package com.tsb.most.basebiz.parm.codes;

import com.tsb.most.framework.bizparm.BaseBizParm;

public class SearchCountryCodeParm extends BaseBizParm{
	private String cntryCd;
	private String cntryNm;
	private String isEuro;
	private String flagState;
	
	public String getCntryCd() {
		return cntryCd;
	}
	public void setCntryCd(String cntryCd) {
		this.cntryCd = cntryCd;
	}
	public String getCntryNm() {
		return cntryNm;
	}
	public void setCntryNm(String cntryNm) {
		this.cntryNm = cntryNm;
	}
	public String getIsEuro() {
		return isEuro;
	}
	public void setIsEuro(String isEuro) {
		this.isEuro = isEuro;
	}
	public String getFlagState() {
		return flagState;
	}
	public void setFlagState(String flagState) {
		this.flagState = flagState;
	}

}
